models/public/single-human-pose-estimation-0001/model.yml

```diff
@@ -0,0 +1,52 @@
+# Copyright (c) 2019 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+description: >-
+
+  The pose may contain up to 17 keypoints: ears, eyes, nose,
+  shoulders, elbows, wrists, hips, knees and ankles.
+
+  The model input is a blob that consists of one images of "1x3x384x288" in
+  RGB order.
+  The model outputs one blob: heatmaps ( shape of output blob "1x17x48x36")
```

Comment:
I don't think this kind of detail needs to be here. It can stay in the Markdown description. (It's also wrong, since the converted model has BGR input.)